### PR TITLE
Adding CONTRIBUTING.md file

### LINKED ISSUE #1065: Missing CONTRIBUTING.md file
All K8s subrepositories should have a CONTRIBUTING.md file, which at the minimum should point to https://github.com/kubernetes/community/blob/master/contributors/guide/README.md. Care should be taken that all information is in sync with the contributor guide.

Subrepositories may also have contributing guidelines specific to that repository. They should be explicitly documented and explained in the CONTRIBUTING.md

Ref:  https://github.com/kubernetes/community/issues/1832

### PR DESCRIPTION
**Pull request by Rajat-0**
Adding the missing Contribution guide.
fixes #1065

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Autoscaler collaborators may add "LGTM" (Looks Good To Me) or an equivalent comment to indicate 
+that a PR is acceptable. Any change requires at least one LGTM. No pull requests can be merged 
+until at least one Ingress collaborator signs off with an LGTM.
+
+### Support Channels
+
+Whether you are a user or contributor, official support channels include:
+
+ * GitHub issues: https://github.com/kubernetes/autoscaler/issues
+ * Slack: kubernetes-users room in the [Kubernetes Slack](https://kubernetes.slack.com/?redir=%2Fmessages%2Fsig-autoscaling)
```
Comment (aleksandra-malinowska) [inline CONTRIBUTING.md]: + #sig-autoscaling channel

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+All changes must be code reviewed. Coding conventions and standards are explained in the official 
+[developer docs](https://github.com/kubernetes/community/tree/master/contributors/devel). Expect 
+reviewers to request that you avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments)
+in your PRs.
+
+### Merge Approval
+
+Autoscaler collaborators may add "LGTM" (Looks Good To Me) or an equivalent comment to indicate 
+that a PR is acceptable. Any change requires at least one LGTM. No pull requests can be merged 
+until at least one Ingress collaborator signs off with an LGTM.
```
Comment (aleksandra-malinowska) [inline CONTRIBUTING.md]: s/Ingress/Autoscaler?

**Comment by Rajat-0** [comment]
@aleksandra-malinowska I have updated the changes as per your comments

**Comment by aleksandra-malinowska** [comment]
/lgtm

Thanks for contributing this!

**Pull request merged**